linux/deploy_vm/rebuild_ubuntu_unattend_install_iso.yml

```diff
@@ -24,6 +24,41 @@
   ignore_errors: true
   register: extract_bios_cfg
 
+- name: "Get Release file path from md5sum.txt"
+  ansible.builtin.shell: "grep '/dists/.*/Release$' '{{ src_iso_dir_path }}/md5sum.txt' | awk '{print $NF}' | sed 's#^\\./##'"
+  register: release_file_path_result
+  ignore_errors: true
+
+- name: "Check Release file path"
+  ansible.builtin.assert:
+    that:
+      - release_file_path_result.stdout is defined
+      - release_file_path_result.stdout
+    fail_msg: "Not found Release file path in md5sum.txt from the ISO image"
+
+- name: "Extract Release file from ISO"
+  community.general.iso_extract:
+    image: "{{ src_iso_file_path }}"
+    dest: "{{ src_iso_dir_path }}"
+    files:
+      - "{{ release_file_path_result.stdout }}"
+
+- name: "Get architectures from Release file"
+  ansible.builtin.shell: "grep '^Architectures:' {{ src_iso_dir_path }}/Release | awk '{print $NF}'"
+  register: ubuntu_arch_result
+  ignore_errors: true
+
+- name: "Check architectures in Release file"
+  ansible.builtin.assert:
+    that:
+      - ubuntu_arch_result.stdout is defined
+      - ubuntu_arch_result.stdout
+    fail_msg: "Not found architectures in file {{ release_file_path_result.stdout }}"
+
+- name: "Set fact of Ubuntu OS architecture"
+  ansible.builtin.set_fact:
+    ubuntu_arch: "{{ ubuntu_arch_result.stdout }}"
```

Comment:
Here can change `ubuntu_arch` to something like `ubuntu_grub_opts`:
```
ubuntu_grub_opts: "{{ (ubuntu_arch_result.stdout == 'arm64') | ternary('', 'console=ttyS0,115200n8') }}"
```
Then below replace values can be changed to:
```
replace: "\\1autoinstall \\2 {{ubuntu_grub_opts}}  "
```